src/main/java/org/betonquest/betonquest/conversation/SlowTellrawConvIO.java

```diff
@@ -58,11 +74,14 @@ public void run() {
                     displayText();
 
                     // Display endLines
-                    for (final String message : endLines) {
-                        SlowTellrawConvIO.super.print(message);
+                    if (endLines != null) {
+                        for (final String message : endLines) {
+                            SlowTellrawConvIO.super.print(message);
+                        }
                     }
```

Comment:
I think that is a good reason to add a debug log.

Reply:
I'm sorry but didn't understand this, I checked its super classes and didn't get any clues, and I may think it's not required.

Reply:
After a short discussion about (missing) logging in Conversation I still say a debug log there would be nice but that will be part of another PR which will specific address that issue.